src/unittests/actorImport/characterImporter/sr5.CharacterImporter.spec.ts

```diff
@@ -26,37 +26,70 @@ export const characterImporterTesting = (context: QuenchBatchContext) => {
             const character = await testActor.create({ 'type': 'character', 'system.metatype': 'human' });
             await character.createEmbeddedDocuments('Item', [item]);
+            assert.lengthOf(character.items, 1);
+
+            await new CharacterImporter().importChummerCharacter(character, {}, importOptions);
 
             assert.lengthOf(character.items, 1);
-            // @ts-expect-error
             assert.strictEqual(character.items.contents[0].name, item.name);
-            // @ts-expect-error
             assert.strictEqual(character.items.contents[0].type, item.type);
         });
 
-        it('Clears all items no actions present', async () => {
-            const item = await testItem.create({ type: 'weapon' });
+        it('Clears all imported items', async () => {
+            const item = await testItem.create({ type: 'weapon' }) as SR5Item;
+            await item.update({'system.importFlags.isImported': true})
+
             const character = await testActor.create({ 'type': 'character', 'system.metatype': 'human' });
             await character.createEmbeddedDocuments('Item', [item]);
+
+            assert.lengthOf(character.items, 1);
+
             await new CharacterImporter().importChummerCharacter(character, chummerFile, importOptions);
 
             assert.isEmpty(character.items);
         });
 
+        it('Clears all items but not imported ones', async () => {
+            const item = await testItem.create({ type: 'weapon' }) as SR5Item;
+
+            const character = await testActor.create({ 'type': 'character', 'system.metatype': 'human' });
+            await character.createEmbeddedDocuments('Item', [item]);
+
+            assert.lengthOf(character.items, 1);
+
+            await new CharacterImporter().importChummerCharacter(character, chummerFile, importOptions);
+
+            assert.lengthOf(character.items, 1);
+        });
+
         it('Clears all items but actions', async () => {
-            const item = await testItem.create({ type: 'action' });
+            const item = await testItem.create({ type: 'action' }) as SR5Item;
             const character = await testActor.create({ 'type': 'character', 'system.metatype': 'human' });
             await character.createEmbeddedDocuments('Item', [item]);
+            
+            assert.lengthOf(character.items, 1);
+
             await new CharacterImporter().importChummerCharacter(character, chummerFile, importOptions);
 
             assert.lengthOf(character.items, 1);
-            // @ts-expect-error
             assert.strictEqual(character.items.contents[0].name, item.name);
-            // @ts-expect-error
             assert.strictEqual(character.items.contents[0].type, item.type);
         });
+
+        // it('Clears all items but effects', async () => {
+        //     let item = await testItem.create({ type: 'weapon' }) as Item;
+        //     item.createEmbeddedDocuments('effects', [{'effects': [{name:'test'}]}])
```

Comment:
Check https://github.com/SR5-FoundryVTT/SR5-FoundryVTT/blob/master/src/unittests/sr5.ActiveEffect.spec.ts
EmbeddedDocuments type is always singular and for effects is `ActiveEffect`. Data provided also isn't in relation of the owning document (`{'effects': ...}) but effect data directly.